### PR TITLE
Last 10 streetcode cards displaying on main page

### PR DESCRIPTION
**Pull request by FlSlaves**
## Summary of change

[Issue](https://github.com/orgs/ita-social-projects/projects/21/views/1?filterQuery=fls&pane=issue&itemId=48642644)

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: Streetcode/Streetcode.BLL/MediatR/Streetcode/Streetcode/GetAllMainPage/GetAllStreetcodesMainPageHandler.cs
Comment (Oleh-Kril) [inline Streetcode/Streetcode.BLL/MediatR/Streetcode/Streetcode/GetAllMainPage/GetAllStreetcodesMainPageHandler.cs:1]: The name of handler does not display the new logic

File: Streetcode/Streetcode.BLL/MediatR/Streetcode/Streetcode/GetPageMainPage/GetPageOfStreetcodesMainPageHandler.cs [hunk 1]
```diff
@@ -33,8 +34,8 @@ public async Task<Result<IEnumerable<StreetcodeMainPageDTO>>> Handle(GetPageOfSt
             var random = new Random(request.shuffleSeed);
             var streetcodes = (await _repositoryWrapper.StreetcodeRepository.GetAllAsync(
                 predicate: sc => sc.Status == DAL.Enums.StreetcodeStatus.Published,
-                include: src => src.Include(item => item.Text).Include(item => item.Images).ThenInclude(x => x.ImageDetails)))
-                .OrderBy(sc => random.Next());
+                include: src => src.Include(item => item.Text).Include(item => item.Images).ThenInclude(x => x.ImageDetails))).
+                OrderByDescending(sc => sc.UpdatedAt).Take(10);
```
Comment (Oleh-Kril) [inline Streetcode/Streetcode.BLL/MediatR/Streetcode/Streetcode/GetPageMainPage/GetPageOfStreetcodesMainPageHandler.cs:38]: I want to pass the number from frontend, don't use 10 everywhere please

**Review by Oleh-Kril — CHANGES_REQUESTED** [review]
Fix comments

**Review by malyna2 — APPROVED** [review]
Сonsider Oleg's comments, then it would be ok

File: Streetcode/Streetcode.BLL/MediatR/Streetcode/Streetcode/GetAllMainPage/GetAllStreetcodesMainPageHandler.cs [hunk 2]
```diff
@@ -42,10 +42,7 @@ public async Task<Result<IEnumerable<StreetcodeMainPageDTO>>> Handle(GetAllStree
                     streetcode.Images = streetcode.Images.Where(x => x.ImageDetails.Alt.Equals(keyNumOfImageToDisplay.ToString())).ToList();
                 }
 
-                var random = new Random();
-                var shuffledStreetcodes = streetcodes.OrderBy(sc => random.Next());
-
-                return Result.Ok(_mapper.Map<IEnumerable<StreetcodeMainPageDTO>>(shuffledStreetcodes));
+                return Result.Ok(_mapper.Map<IEnumerable<StreetcodeMainPageDTO>>(streetcodes).OrderByDescending(c => c.UpdatedAt).Take(10));
```
Comment (NadiaKishchuk) [inline Streetcode/Streetcode.BLL/MediatR/Streetcode/Streetcode/GetAllMainPage/GetAllStreetcodesMainPageHandler.cs:45]: I believe this is better to create the constant variable for that 10

**Pull request closed without merging**